Traffic Stats error when trying to get LastSummaryTime

When querying Traffic Ops to get LastSummaryTime, Traffic Stats is throwing an error like: 
[ERROR] 2018-01-30 22:33:40 parsing time "2018-01-30 17:31:40-05" as "2006-01-02 15:04:05+00": cannot parse "-05" as "+00".  
Seems like a simple date format change would fix this issue.

--- a/traffic_stats/traffic_stats.go
+++ b/traffic_stats/traffic_stats.go
@@ -506,7 +506,7 @@ func getToData(config StartupConfig, init bool, configChan chan RunningConfig) {
 	if err != nil {
 		errHndlr(err, ERROR)
 	} else {
-		lastSummaryTime, err := time.Parse("2006-01-02 15:04:05-05", lastSummaryTimeStr)
+		lastSummaryTime, err := time.Parse("2006-01-02 15:04:05-07", lastSummaryTimeStr)
 		if err != nil {
 			log.Error("Error parsing lastSummaryTime: " + err.Error())
 		} else {

--- a/traffic_stats/traffic_stats.go
+++ b/traffic_stats/traffic_stats.go
@@ -506,9 +506,9 @@ func getToData(config StartupConfig, init bool, configChan chan RunningConfig) {
 	if err != nil {
 		errHndlr(err, ERROR)
 	} else {
-		lastSummaryTime, err := time.Parse("2006-01-02 15:04:05+00", lastSummaryTimeStr)
+		lastSummaryTime, err := time.Parse("2006-01-02 15:04:05-05", lastSummaryTimeStr)
 		if err != nil {
-			errHndlr(err, ERROR)
+			log.Error("Error parsing lastSummaryTime: " + err.Error())
 		} else {
 			runningConfig.LastSummaryTime = lastSummaryTime
 		}

